### DISCUSSION TITLE
Uploading db via POST request

### DISCUSSION THREAD

**Discussion by dev3rsan** [cpvalente/ontime#660] [General]
Hi, I have schedule in automation system and convert it to rundown,json. Manually it` work great but I want to put it to DB by script. Can I put db.json file via some POST request?
 I try it like "curl --location --request POST localhost:4001/ontime/db -H 'Content-Type: application/json' -d @rundown.json " but {"message":"File not found"} it`s only answer.

**Comment by cpvalente** [comment]
Hi @dev3rsan, thank you for reaching out

I suspect you are finding issues with the multipart post
see the docs for curl here https://everything.curl.dev/http/multipart

I have never used a curl command to upload this type of payload, but I am happy to help
